### PR TITLE
Vue sort preference

### PR DESCRIPTION
**Pull request by arnaudroystatcan**
Fix JIRA DI-87
- Add single draggable list vue component
- Add two draggable list vue component
- Add cypress tests

### CODE REVIEW

File: src/Themes/HackathonTheme/HackathonTheme.csproj [hunk 1]
```diff
@@ -6,6 +6,10 @@
     <CodeAnalysisRuleSet>..\..\..\roslynator.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="C:\Users\Arnaud Roy\.nuget\packages\orchardcore.module.targets\1.0.0-rc2-16105\build\Package.Build.props" />
```
Comment (jptissot) [inline src/Themes/HackathonTheme/HackathonTheme.csproj:10]: Remove this group

**Comment by jptissot** [comment]
If you could fix the test, otherwise it looks good.

File: src/Themes/VuetifyTheme/Assets.json [hunk 2]
```diff
@@ -12,5 +12,12 @@
       "node_modules/vue/dist/vue.min.js"
     ],
     "output": "wwwroot/Scripts/vue.min.js"
+  },
+    {
+    "copy": true,
+    "inputs": [
+      "node_modules/vuedraggable/dist/vuedraggable.umd.*"
+    ],
+    "output": "wwwroot/Scripts/@"
```
Comment (jptissot) [inline src/Themes/VuetifyTheme/Assets.json:21]: Is this required ?

Comment (arnaudroystatcan) [reply src/Themes/VuetifyTheme/Assets.json]: removed

**Review by jptissot — APPROVED** [review]

**Pull request merged**